client/client.go

```diff
@@ -200,7 +200,11 @@ func (client *Client) ServerVersion() (string, error) {
 		version = "unknown"
 	}
 
-	return fmt.Sprintf("%s (series %s)", version, sysInfo.Series), nil
+	if sysInfo.OnClassic {
+		return fmt.Sprintf("%s (series %s, os: %s, classic)", version, sysInfo.Series, sysInfo.OS), nil
```

Comment:
What do we do on an all-snap image? I would perhaps just not print ubuntu 16.04 line then.